tools/cldr-apps/src/main/java/org/unicode/cldr/web/SurveyTool.java

```diff
@@ -384,80 +209,29 @@ private void includeCss(HttpServletRequest request, PrintWriter out) {
      * @throws JSONException
      */
     public static void includeJavaScript(HttpServletRequest request, Writer out) throws IOException, JSONException {
-        if (useDojo(request)) {
-            includeDojoJavaScript(out);
-        } else {
-            // Load the big bundle
-            out.write("<script src=\"dist/bundle" + getCacheBustingExtension(request) + ".js\"></script>\n");
-        }
+        // Load the big bundle
+        out.write("<script src=\"dist/bundle" + getCacheBustingExtension(request) + ".js\"></script>\n");
         includeJqueryJavaScript(request, out);
         includeCldrJavaScript(request, out);
     }
 
-    private static void includeDojoJavaScript(Writer out) throws IOException {
-        out.write("<script>dojoConfig = {parseOnLoad: false, async: true,};</script>\n");
-        out.write("<script src='//ajax.googleapis.com/ajax/libs/dojo/1.14.1/dojo/dojo.js'></script>\n");
-    }
-
     private static void includeJqueryJavaScript(HttpServletRequest request, Writer out) throws IOException {
-        // For compatibility with old Dojo, use old jquery, otherwise use newest jquery
         // Per https://en.wikipedia.org/wiki/JQuery#Release_history --
-        // jquery 1.11: January 24, 2014
         // jquery 3.5.1: May 4, 2020
-        final boolean doUseDojo = useDojo(request);
-        if (doUseDojo) {
-            out.write("<script src='//ajax.googleapis.com/ajax/libs/jquery/1.11.0/jquery.min.js'></script>\n");
-        } else {
-            out.write("<script src='//ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js'></script>\n");
-        }
+        out.write("<script src='//ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js'></script>\n");
```

Comment:
yeah… we should fold this into the bundle.

That works easily with jQuery, a little harder with jquery UI for some reason.

Reply:
yes